pkgs/development/python-modules/laszip/default.nix

```diff
@@ -0,0 +1,59 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, scikit-build-core
+, distlib
+, pytestCheckHook
+, pyproject-metadata
+, pathspec
+, pybind11
+, cmake
+, LASzip
+}:
+
+buildPythonPackage rec {
+  pname = "laszip-python";
+  version = "0.2.1";
+
+  format = "pyproject";
+
+  src = fetchFromGitHub {
+    owner = "tmontaigu";
+    repo = pname;
+    rev = version;
+    sha256 = "sha256-ujKoUm2Btu25T7ZrSGqjRc3NR1qqsQU8OwHQDSx8grY=";
+  };
+
+  nativeBuildInputs = [
+    scikit-build-core
+    scikit-build-core.optional-dependencies.pyproject
+    cmake
+  ];
+
+  buildInputs = [
+    pybind11
+    LASzip
+  ];
+
+  checkInputs = [
+    pytestCheckHook
+  ];
+
+  preBuild = ''
+    cd ..
+  '';
+
+  # There are no tests, and pytest causes stdenv setup script to crash in the
+  # case of no tests here
+  doCheck = false;
+
+  pythonImportsCheck = [ "laszip" ];
+
+  meta = with lib; {
+    license = licenses.mit;
```

Comment:
Not that this really matters, but could you move it below `description` and `homepage` like the other package in this PR does?